src/main/java/io/orangebeard/client/OrangebeardV2Client.java

```diff
@@ -116,6 +117,20 @@ public void log(Log log) {
         }
     }
 
+    @Override
+    public void log(Set<Log> logs) {
```

Comment:
Should we let the `log(Log log)` function just call this one using `log(Collections.singleton(log)`? Any reason to keep both explicit calls to the /log endpoint?

Reply:
Good point.